<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TDialogData</class>
 <widget class="QDialog" name="TDialogData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,2,1">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>工      号</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>工       资</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="dbComboDep">
           <item>
            <property name="text">
             <string>销售部</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>技术部</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>行政部</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="dbComboSex">
           <item>
            <property name="text">
             <string>男</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>女</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>未知</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>部       门</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>姓      名</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>性      别</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>出生省份</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="dbComboProvince">
           <item>
            <property name="text">
             <string>北京</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>河北</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>湖南</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>重庆</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>安徽</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>上海</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>湖北</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="dbEditName"/>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="dbSpinEmpNo">
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSpinBox" name="dbSpinSalary">
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>出生日期</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDateEdit" name="dbEditBirth"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>照    片</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="LabPhoto">
            <property name="text">
             <string>dbLabPhoto</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnInput">
           <property name="text">
            <string>导入照片</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/00.JPG</normaloff>:/images/00.JPG</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClearPhoto">
           <property name="text">
            <string>清除照片</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/103.bmp</normaloff>:/images/103.bmp</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnOK">
           <property name="text">
            <string>确    定</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/704.bmp</normaloff>:/images/704.bmp</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClose">
           <property name="text">
            <string>取    消</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/706.bmp</normaloff>:/images/706.bmp</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>备    注</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPlainTextEdit" name="editMemo"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>TDialogData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>TDialogData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
